Fails to build, implicit-function-declaration

### What Happened?

Introduced in be8b8a3a6003440647227c08ae96250d1ffb9134.  Current head affected (0c12d796ac9f11828d33a354dcb9597e4d45b8ef).
```
src/gala.p/WindowManager.c:2610:9: error: implicit declaration of function ‘atk_bridge_adaptor_init’ [-Wimplicit-function-declaration]
 2610 |         atk_bridge_adaptor_init (0, _tmp1_, (gint) 0);
      |         ^~~~~~~~~~~~~~~~~~~~~~~
```

Compiler info:
```
C compiler for the host machine: cc (gcc 14.2.1 "cc (GCC) 14.2.1 20240805")
C linker for the host machine: cc ld.bfd 2.43.0
Vala compiler for the host machine: valac (valac 0.56.17)
```

### Steps to Reproduce

Clone repo.  Configure and build with meson.  See error.

### Expected Behavior

Should build without error.  Commit 70e4642f8911cdee4cde941f6ec817971e947aa3 built successfully.

### OS Version

Other Linux

### Software Version

Compiled from git

### Log Output

_No response_

### Hardware Info

_No response_

--- a/src/WindowManager.vala
+++ b/src/WindowManager.vala
@@ -388,7 +388,9 @@ namespace Gala {
                 return;
             }
 
-            AtkBridge.adaptor_init (0, {});
+            string[] args = {};
+            unowned string[] _args = args;
+            AtkBridge.adaptor_init (ref _args);
         }
 
         private void update_ui_group_size () {

--- a/vapi/atk-bridge-2.0.vapi
+++ b/vapi/atk-bridge-2.0.vapi
@@ -1,6 +1,6 @@
-[CCode (lower_case_cprefix = "atk_bridge_")]
+[CCode (cheader_filename = "atk-bridge.h", lower_case_cprefix = "atk_bridge_")]
 namespace AtkBridge {
-	public static int adaptor_init (int argc, char[] argv);
+    public static int adaptor_init ([CCode (array_length_pos = 0.9)] ref unowned string[] argv);
     public static void adaptor_cleanup ();
     public static void set_event_context (GLib.MainContext cnx);
 }

